examples/llm_serving/model/wrapper.py

```diff
@@ -6,13 +6,10 @@
 import warnings
 
 import alpa
+from alpa import timers
```

Comment:
Rebase and revert all debugging changes. The lastest version fixed the import problem with opt_1d.

Reply:
Finished in the latest commits. I add the change of pad_token_id in a if statement when model is bloom.